### PR TITLE
Add new images

### PR DESCRIPTION
**Pull request by mmitche**

### CODE REVIEW

**Comment by mmitche** [comment]
@VSadov This will be the image when it finishes copying

**Comment by VSadov** [comment]
CC: @jaredpar

**Comment by mmitche** [comment]
@VSadov don't merge this quite yet

**Comment by VSadov** [comment]
@mmitche  I did not plan to merge since I am a bit fuzzy on what effects that might have :)

File: jobs/generation/Utilities.groovy [hunk 1]
```diff
@@ -255,6 +257,8 @@ class Utilities {
                                 'latest-or-auto-dev15-internal':'win2016-20170214-internal',
                                 // For internal runs - Win2016 + VS15.0
                                 'latest-or-auto-dev15-0-internal':'win2016-20170307-internal',
+                                // For internal runs - Win2016 + VS15.1
+                                'latest-dev15-1-internal':'win2016-20170427-internal',
```
Comment (jaredpar) [inline jobs/generation/Utilities.groovy]: What is the difference on internal runs?

File: management/nodes/startupScripts/win2016-20170427.ps1 [hunk 2]
```diff
@@ -0,0 +1,64 @@
+Set-ExecutionPolicy Unrestricted
```
Comment (jaredpar) [inline management/nodes/startupScripts/win2016-20170427.ps1]: Is this just boiler plate code?  Not sure I can offer much reviwing it.

**Comment by jaredpar** [comment]
Can we merge this?  Need to start validating these images.

**Comment by mmitche** [comment]
@jaredpar I'm updating the CI servers now.  Going to add a couple more images to this.  Will merge in a few.

File: jobs/generation/Utilities.groovy (hunk 1 above)
Comment (mmitche) [inline jobs/generation/Utilities.groovy]: Nothing, just an isolated pool

**Pull request merged**